services/connectivity/messaging/src/main/java/org/eclipse/ditto/services/connectivity/messaging/amqp/AmqpSpecificConfig.java

```diff
@@ -148,8 +148,8 @@ private static void addCredentials(final LinkedHashMap<String, String> parameter
         final var password = connection.getPassword();
         if (username.isPresent() && password.isPresent()) {
             // URL-decode the username and password to preserve previous behavior (no apparent encoding of parameters)
```

Comment:
This comment above is no longer valid, is it?

Reply:
oops, no it isn't. Will remove it.